src/server/db/schema.ts

```diff
@@ -310,6 +310,7 @@ export const userType = pgEnum("user_type", ["hacker", "organizer", "sponsor"]);
 export const users = createTable("user", {
   id: varchar("id", { length: 255 }).notNull().primaryKey(),
   name: varchar("name", { length: 255 }),
+  password: varchar("password", { length: 255 }),
```

Comment:
I forgot about this: whenever we make a schema change we have to run `npm run db:generate` to create the migration SQL scripts (basically a SQL script that changes the db's schema without deleting data)

Reply:
This is what's currently failing the tests 👍